Code/Tools/FBuild/FBuildCore/FBuildOptions.cpp

```diff
@@ -173,6 +173,26 @@ FBuildOptions::OptionsResult FBuildOptions::ProcessCommandLine( int argc, char *
                 m_Args += argv[ sizeIndex ];
                 continue;
             }
+            else if (thisArg == "-distcompressionlevel")
```

Comment:
The -help output needs to be updated for this new option at the bottom of this file.